### PR TITLE
theatre_reconcile_errors_total

### PR DESCRIPTION
**Pull request by lawrencejones**
Add a metric that tracks the number of reconcile errors we get from
theatre. This enables us to alert whenever theatre is failing to
reconcile objects.

### CODE REVIEW

File: cmd/helpers.go [hunk 1]
```diff
@@ -42,7 +49,16 @@ func (opt *commonOptions) Logger() kitlog.Logger {
...
+	logger = kitlog.With(logger, "ts", kitlog.DefaultTimestampUTC, "caller", logging.RecorderAwareCaller())
+
+	stdlog.SetOutput(kitlog.NewStdlibAdapter(logger))
+	klog.SetOutput(kitlog.NewStdlibAdapter(logger))
 
 	return logger
 }
+
+func (opt *commonOptions) ListenAndServeMetrics(logger kitlog.Logger) {
+	logger.Log("event", "metrics.listen", "address", opt.MetricAddress, "port", opt.MetricPort)
```
Comment (lawrencejones) [inline cmd/helpers.go]: `metrics_listen`

File: cmd/rbac-manager/main.go [hunk 2]
```diff
@@ -31,30 +27,29 @@ var (
 	app     = kingpin.New("rbac-manager", "Manages rbac.crd.gocardless.com resources").Version(Version)
 	refresh = app.Flag("refresh", "Refresh interval checking directory sources").Default("1m").Duration()
 
+	common = cmd.NewCommonOptions(app)
```
Comment (lawrencejones) [inline cmd/rbac-manager/main.go]: commonOpts

**Comment by rnaveiras** [comment]
@lawrencejones it's ready now and deployed to the lab. Do you mind take a final look?

https://prometheus-lab.gocardless.io/graph?g0.range_input=1h&g0.expr=rate(theatre_reconcile_errors_total%5B5m%5D)&g0.tab=0

**Comment by lawrencejones** [comment]
Lgtm!

**Pull request merged**